README.md

```diff
@@ -1 +1,51 @@
-# cr-review-hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> Learning best practices of GitHub flow.
+
+![screenshot](./app_screenshot.png)
+
+Hello Microverse
+
+## Built With
+
+- HTML & CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] \[OPTIONAL\] It would be great to have instructions to set up this project.